src/croniter/croniter.py

```diff
@@ -633,6 +664,8 @@ def _get_prev_nearest_diff(self, x, to_check, range_val):
                 return d - x
         if 'l' in candidates:
             return -x
+        if range_val is None:
```

Comment:
also add documentation comment

Reply:
I added documents to both functions. You can edit them however you want.
I'm not good at writing documents.

Reply:
i mean comments, document what you did, and why

Reply:
Just move a part f the docstring as a comment where it's needed, it will be good (i can do it myself, but i prefer you to do it to keep paternity :) )


i mean:

change
```python

    def _get_prev_nearest_diff(self, x, to_check, range_val):
        """
        `range_val` is the range of a field.
        If no available time, we can move to previous loop(like previous month).
        Range_val can also be set to `None` to indicate that there is no loop.
        Currently should only used for `year` field
        When range_val is None and x not exists in to_check,
        `None` will be returned to suggest no more available time
        """
        candidates = to_check[:]
        candidates.reverse()
        for d in candidates:
            if d != 'l' and d <= x:
                return d - x
        if 'l' in candidates:
            return -x
        if range_val is None:

```

to 

```python


    def _get_prev_nearest_diff(self, x, to_check, range_val):
        """
        `range_val` is the range of a field.
        If no available time, we can move to previous loop(like previous month).
        Range_val can also be set to `None` to indicate that there is no loop.
        ( Currently should only used for `year` field )
        """
        candidates = to_check[:]
        candidates.reverse()
        for d in candidates:
            if d != 'l' and d <= x:
                return d - x
        if 'l' in candidates:
            return -x

        # If no available time, we can move to previous loop (like previous month).
        # Range_val can also be set to `None` to indicate that there is no loop.
        # Currently should only used for `year` field
        # When range_val is None and x not exists in to_check,
        # `None` will be returned to suggest no more available time
        if range_val is None:
```

Reply:
thanks for your detailed suggestions